features/step_definitions/consultation_steps.rb

```diff
@@ -101,19 +101,19 @@
   expect(expected_number_of_attachments.to_i).to eq(Response.last.attachments.count)
 end
 
-When(/^I set the order of the responses attachments to:$/) do |attachment_order|
-  attachment_order.hashes.each do |attachment_info|
-    attachment = Attachment.find_by(title: attachment_info[:title])
-    fill_in "ordering[#{attachment.id}]", with: attachment_info[:order]
-  end
-  click_on "Save attachment order"
-end
-
-Then(/^the responses attachments should be in the following order:$/) do |attachment_list|
-  attachment_ids = all(".existing-attachments > li").map { |element| element[:id] }
-
-  attachment_list.hashes.each_with_index do |attachment_info, index|
-    attachment = Attachment.find_by(title: attachment_info[:title])
-    expect("attachment_#{attachment.id}").to eq(attachment_ids[index])
-  end
-end
+# When(/^I set the order of the responses attachments to:$/) do |attachment_order|
```

Comment:
I'm not sure how to deal with these tests in the design system scenario where there is no ordering of attachments present.

Reply:
Cool. We've got prior art for this here 

https://github.com/alphagov/whitehall/blob/main/features/step_definitions/attachment_steps.rb#L52-L81

It might be that you can literally just use the attachment methods as i think they do the exact same thing to the last couple of blocks in the feature spec would be 

```
  When I set the order of attachments to:
      | title                    | order |
      | Beard Length Graphs 2012 | 0     |
      | Outcome attachment title | 1     |
    Then the attachments should be in the following order:
      | title                    |
      | Beard Length Graphs 2012 |
      | Outcome attachment title |
```